docs/development.md

```diff
@@ -179,6 +180,27 @@ To "log out", submit the following API call:
 POST http://localhost:8080/_ah/login?action=Log+Out
 ```
 
+## Setting up reCAPTCHA
```

Comment:
For something that is used only in one page and is only applicable in production, mentioning here is an overkill.

You should use a separate file for it, [this is one such example](https://github.com/TEAMMATES/teammates/blob/master/docs/emails.md). If you don't know the existence of that file, then it's a success sign, because 99% developers need not be concerned about it.